### ISSUE TITLE
Assembly error when importing Nakama

### ISSUE THREAD

**Issue by mnarimani** [heroiclabs/nakama-unity#78]
I used the released package on github page in Unity 2019.2.3.
But I'm receiving this error:
```
Error: Could not load signature of Nakama.ClientExtensions:NewSocket due to: Could not resolve type with token 01000007 (from typeref, class/assembly Nakama.ISocket, Nakama, Version=2.3.1.0, Culture=neutral, PublicKeyToken=null) assembly:Nakama, Version=2.3.1.0, Culture=neutral, PublicKeyToken=null type:Nakama.ISocket member:(null) signature:<none>
```
How can I solve this?

**Comment by novabyte** [comment]
Hey @mnarimani I'll grab Unity 2019.2.3 and see what happens when the SDK is loaded.

**Comment by mnarimani** [comment]
@novabyte I found what was wrong with it. 
I'm using assembly definition files for my project. 
3rd party libraries also need to use assembly definition files if you want to reference them. So I added an assembly definition file named "Nakama". 
That was the problem. There is a dll called "Nakama" inside the package. And compiler didn't know which Nakama assembly was correct.
I renamed the assembly definition file and error went away.
